server/i18n/en.json

```
@@ -6063,6 +6067,10 @@
     "id": "app.plugin.signature_decode.app_error",
     "translation": "Unable to decode base64 signature."
   },
+  {
+    "id": "app.plugin.skip_installation.app_error",
+    "translation": "Skipping installation of plugin {{.Id}} since existing version is newer."
```

1/5 on the suggested wording.

If we try to install the same version, the error will be raised telling us that the existing one is newer (but it's the same). It would be nice to have a more accurate message here and in the mlog line.

```suggestion
    "translation": "Skipping installation of plugin {{.Id}} since existing version is equal or newer."
```